src/main/java/org/jawk/backend/AVM.java

```diff
@@ -1612,7 +1612,7 @@ public void interpret(AwkTuples tuples)
 						// stack[1] = second actual parameter
 						// etc.
 						Address func_addr = position.addressArg();
-						String func_name = position.arg(1).toString();
+						//String func_name = position.arg(1).toString();

```

Comment:
Let's make it that we delete code that we don't want to be used anymore and not just comment it out :)